src/button-group/interfaces.ts

```diff
@@ -0,0 +1,121 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import { ButtonDropdownProps } from '../button-dropdown/interfaces';
+import { IconProps } from '../icon/interfaces';
+import { BaseComponentProps } from '../internal/base-component';
+import { NonCancelableEventHandler } from '../internal/events';
+import { InternalBaseComponentProps } from '../internal/hooks/use-base-component';
+
+export interface ButtonGroupProps extends BaseComponentProps {
+  /**
+   * Adds `aria-label` to the button group toolbar element.
```

Comment:
Let's add a sentence about when to use this optional property as we have it in other components. Suggestion:
Use to assign unique labels when there are multiple button groups on one page.

Reply:
I cahnged it to:
  /**
   * Adds `aria-label` to the button group toolbar element.
   * Use this to provide a unique accessible name for buttons groups on the page.
   */